tests/phpunit/tests/test-links-subdomain-sub-dir.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class Links_Subomain_Sub_Dir_Test extends PLL_Domain_UnitTestCase {
+	use PLL_Directory_Trait;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->hosts = array(
+			'en' => 'http://example.org',
+			'fr' => 'http://fr.example.org',
+			'de' => 'http://de.example.org',
+		);
+
+		self::$model->options['hide_default'] = 1;
+		self::$model->options['force_lang'] = 2;
+		self::$model->options['domains'] = $this->hosts;
```

Comment:
Unfortunately it's not as easy as it seems. I tried and it raised a bunch of errors...
Since static model was previously used in these test class, I propose to handle this in another PR (because it might need some investigation).
Maybe it can be done at the same time as https://github.com/polylang/polylang-pro/issues/1703?